app/decorators/course_decorator.rb

```diff
@@ -130,6 +130,27 @@ def sorted_subjects
     object.subjects.map(&:subject_name).sort.join("<br>").html_safe
   end
 
+  def chosen_subjects(course)
+    if course.master_subject_id.nil?
+      sorted_subjects
+    else
+      main_subject_id = course.master_subject_id
```

Comment:
May as well change `master_subject_id` to `main_subject_id`

Reply:
What, you mean rename the column in the db?

Reply:
yes

Reply:
Can do in a follow up PR